docs/reference/cli-reference.md

```diff
@@ -348,7 +348,7 @@ infra identities list [flags]
 Delete an identity
 
 ```
-infra identities remove NAME [flags]
+infra identities remove IDENTITY [flags]
```

Comment:
I changed this because as I was writing tests for it I noticed it was inconsistent with the other `infra identities` commands. The usage string for both `add` and `edit` use `IDENTITY` as the name of the positional argument.